### PR TITLE
#1376 Update our Twilio error codes to handle the new 30034

### PR DESCRIPTION
**Pull request by EvanParish**
# Description

This change simply adds a new twilio error code that we were seeing in our logs.

#1376

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## How Has This Been Tested?

- [x] Added the new error code to our unit tests and the tests pass

## Checklist

- [x] Appropriate labels added to this pull request (i.e. "enhancement", "dependencies", etc.)
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ldraney — APPROVED** [review]

File: .talismanrc [hunk 1]
```diff
@@ -1,212 +1,216 @@
 fileignoreconfig:
-    - filename: .github/workflows/staging-ci.yaml
```
Comment (cris-oddball) [inline .talismanrc:2]: I don't think I have ever seen so many talisman changes, what happened?

Comment (EvanParish) [reply .talismanrc:2]: I'm honestly not sure. It did that when I went to commit my changes.

**Review by cris-oddball — DISMISSED** [review]
Please assign yourself to this PR and move your ticket into Dev Testing.
Please also just run all that talisman stuff past someone else on the team - seems like an inordinate number of changes?

File: .talismanrc (hunk 1 above)
Comment (EvanParish) [reply .talismanrc]: I manually reverted talisman and when it ran again it didn't prompt for changes this time. It looks like previously it had messed up the formatting / indentation and added some test files to `.talismanrc`.

**Review by ldraney — APPROVED** [review]

**Pull request merged**